datetimepickeredittext/src/main/java/io/blackbox_vision/datetimepickeredittext/view/DatePickerEditText.kt

```diff
@@ -57,6 +58,9 @@ class DatePickerEditText : AppCompatEditText, OnFocusChangeListener, OnClickList
         inputType = InputType.TYPE_NULL
         onFocusChangeListener = this
         setOnClickListener(this)
+
+        /*Set fragment manager*/
+        manager = (context as AppCompatActivity).supportFragmentManager
```

Comment:
Have you tested that this is running through different `android` versions? My intention for this library, is to be `retro-compatible`.

Reply:
The same goes for the other files which contains this line:

```kotlin
manager = (context as AppCompatActivity).supportFragmentManager
```

Reply:
I tested this on most of the API levels and it's working fine. You can also test this on different versions. 
The main intention of adding this line is that most of the time user added the xml code but forget to add setManager in java after run the app crashed. By this code the app will not crash. I already added this code in all the files.

Reply:
I understand your point, and believe you have tested it. Great!